src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -257,10 +258,12 @@ class AuthScreens extends React.Component {
     }
 
     render() {
-        const screenOptions = getRootNavigatorScreenOptions(this.props.isSmallScreenWidth);
+        const isHomeOdSelected = this.context.selectedSubNavigationOption === SCREENS.HOME_OLDDOT;
```

Comment:
```suggestion
        const isHomeOldDotSelected = this.context.selectedSubNavigationOption === SCREENS.HOME_OLDDOT;
```

I think we should just make the name longer here. It took me a long time to realize `Od === oldDot`

Reply:
`isOldDotHomeSelected` might be even better